src/app/adminLayout/admin-header/admin-header.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+import {MenuItem} from "primeng/api";
+
+@Component({
+  selector: 'app-admin-header',
+  templateUrl: './admin-header.component.html',
```

Comment:
Skoro to element naszge admin-layout - to może warto zdefiniować tutaj template 

```template: '<p-menubar [model]="items"></p-menubar>'```

 i wyrzucić wszystkie pliki oprócz tego componentu, nie wiem jak się odnieść do testów po stronie frontu, czy tutaj jak mamy tylko tak na prawde zdefiniowany template to czy jest potrzebny plik z testami?

Ten projekt rośnie no i trzeba by się zastanowić gdzie ew. będą potrzebne pliki z testami a gdzie nie (osobiście nie czytałem nic na ten temat jeszcze )

Reply:
Ja tam wolę mieć osobnego htmla, bo docelowo pewnie ten header będzie trochę większy niż 1 linia kodu, więc bym nie zmieniał. Co to testów to nie znam się na angularowych testach więc się nie wypowiem.

Reply:
ok